plugins/bash-preexec/README.md

```diff
@@ -0,0 +1,3 @@
+# bash preexec
+
+this allows to set the window title on termux
```

Comment:
This doesn't explain bash-preexec. This is an explanation of lib/xterm.sh. In addition, it is not specific to termux.

Reply:
fixed